android-appcoins-billing/src/main/java/com/appcoins/sdk/android_appcoins_billing/CatapultAppcoinsBilling.java

```diff
@@ -17,10 +19,11 @@
 import com.appcoins.sdk.billing.SkuDetailsParams;
 import com.appcoins.sdk.billing.SkuDetailsResponseListener;
 
-public class CatapultAppcoinsBilling implements AppcoinsBillingClient {
+public class CatapultAppcoinsBilling implements AppcoinsBillingClient , DialogVisibleListener {
```

Comment:
I'm gonna be picky but these names matter to give the user a sense of the class purpose.
I know what you're trying to say but it's ambiguous. Does it listen to the dialog only when it's visible? I went ahead and saw that the answer is no. But the name should tell me this already.
I suggest changing it to IsDialogVisibleListerner or even better DialogVisibilityListener (you'll see why on the next comment)